libraries/AP_BattMonitor/AP_BattMonitor_Params.h

```diff
@@ -25,6 +25,9 @@ class AP_BattMonitor_Params {
         MPPT_Power_Off_At_Boot              = (1U<<4),  // MPPT Disabled at startup (aka boot), if HW supports it
         MPPT_Power_On_At_Boot               = (1U<<5),  // MPPT Enabled at startup (aka boot), if HW supports it. If Power_Off_at_Boot is also set, the behavior is Power_Off_at_Boot
         GCS_Resting_Voltage                 = (1U<<6),  // send resistance resting voltage to GCS
+#ifdef HAL_BUILD_AP_PERIPH
+        Internal_Only_DO_NOT_Broadcast      = (1U<<7),  // INTERNAL USE ONLY. Do not broadcast this instance on the CAN bus.
```

Comment:
I think we should name this something like "Do_Not_Broadcast" .. so drop the "Internal" bit.